roles/network_plugin/contiv/templates/contiv-api-proxy.yml.j2

```diff
@@ -0,0 +1,55 @@
+# This manifest deploys the Contiv API Proxy Server on Kubernetes.
```

Comment:
Is there an upstream source URL for this? Just good to know its location for future maintenance